Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.py

```diff
@@ -231,7 +232,7 @@ def fetch_incidents(client: IMAPClient,
         next_run: This will be last_run in the next fetch-incidents
         incidents: Incidents that will be created in Demisto
     """
-    LOG('in fetch incidents function')
+    demisto.debug('in fetch incidents function')
```

Comment:
use the [logger function](https://github.com/demisto/content/blob/master/Packs/Base/Scripts/CommonServerPython/CommonServerPython.py#L1679) instead